Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -840,6 +928,97 @@ class ASHeaderView: UICollectionReusableView {
     }
 }
 
+class LibraryShortcutView: UIView {
+
+    lazy var button: UIButton = {
+        let button = UIButton()
+        button.backgroundColor = UIColor(rgb: 0xc7eafd)
+        button.setImage(UIImage.templateImageNamed("menu-panel-Bookmarks"), for: .normal)
+        button.tintColor = .white
+        return button
+    }()
+
+    lazy var title: UILabel = {
+        let title = UILabel()
+        return title
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        addSubview(button)
+        addSubview(title)
+        button.snp.makeConstraints { make in
+            make.width.equalTo(self)
+            make.height.equalTo(self.snp.width)
+        }
+        title.adjustsFontSizeToFitWidth = true
+        title.minimumScaleFactor = 0.7
+        title.font = DynamicFontHelper.defaultHelper.SmallSizeRegularWeightAS
+        title.textAlignment = .center
+        title.snp.makeConstraints { make in
+            make.top.equalTo(button.snp.bottom).offset(5)
+            make.leading.trailing.equalToSuperview()
+        }
+        button.imageView?.contentMode = .scaleToFill
+        button.contentVerticalAlignment = .fill
+        button.contentHorizontalAlignment = .fill
+        button.imageEdgeInsets = UIEdgeInsets(equalInset: 15)
+    }
+
+    required init(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func layoutSubviews() {
+        button.layer.cornerRadius = self.frame.width/2
+        super.layoutSubviews()
+    }
+}
+
+
+class ASLibraryCell: UICollectionViewCell {
+
+    var mainView = UIStackView()
+
+    struct LibraryPanel {
+        let title: String
+        let image: UIImage?
+        let color: UIColor
+        //let selector: Selector
+    }
+
+    var libraryButtons: [LibraryShortcutView] = []
+
+    let bookmarks = LibraryPanel(title: "Bookmarks", image: UIImage.templateImageNamed("menu-Bookmark"), color: UIColor(rgb: 0x0A84FF))
```

Comment:
Should this be a themable color?

Reply:
These colors arent 100% I think leaving them here for now is fine. I havent gotten dark theme colors for these buttons yet.